src/default_claims.rs

```diff
@@ -0,0 +1,33 @@
+use axum::response::{IntoResponse, Response};
+use reqwest::StatusCode;
+use serde::{Deserialize, Serialize};
+
+use crate::{ParseTokenClaims, TokenError};
+
+// The specific claims you want to parse from received JWTs.
+#[derive(Deserialize, Serialize)]
+pub struct DefaultClaims {}
+
+impl ParseTokenClaims for DefaultClaims {
+    type Rejection = TokenClaimsError;
+}
```

Comment:
Instead of `DefaultClaims`, I think a name like `EmptyClaims` or something similar would be more explicit as to the purpose of the struct. Also, I think it should have a docstring explaining that it should be used when you just need to verify the audience.

Reply:
Thanks! will change

Reply:
Have been playing a bit, when time..., think I want to use middleware. I think this library should be a s small as possible. Closing this for now. Thanks!

Reply:
I agree with keeping the library small. Thanks for the contributions, and feel free to add a link to your solution if you build something open source.